Expose isOpen field dynamically

tsoliangwu0130 opened the pull request.

tsoliangwu0130 committed:
Expose isOpen dynamically to result

tsoliangwu0130 committed:
Minor change

tsoliangwu0130 committed:
Add isOpen field

magical commented on src/main/groovy/edu/oregonstate/mist/locations/frontend/mapper/LocationMapper.groovy:
```diff
@@ -50,5 +52,28 @@ class LocationMapper {
         ro?.attributes?.remove("hashCode")
+
+        // expose isOpen field
+        def now = DateTime.now(DateTimeZone.UTC)
```
this function would be easier to test if `now` were supplied as an argument

magical replied:
nevermind. thanks for pointing out the existence of the DateTimeUtils.setCurrentMillisFixed function :)

tsoliangwu0130 committed:
Add unit test for isOpen

not using syntax shorthand so that it's eaiser to know each assertion tests which case

tsoliangwu0130 commented:
Not merging this PR for now because of the current caching proxy would affect it. We should merge this PR until we implement some cache-control functionality.
